packages/es-components/src/components/controls/radio-buttons/RadioGroup.js

```diff
@@ -8,10 +8,12 @@ import Fieldset from '../../containers/fieldset/Fieldset';
 import RadioButton from './RadioButton';
 
 const RadioFieldset = styled(Fieldset)`
+  margin-bottom: 20px;
   legend {
     border: none;
     font-size: 22px;
-    margin-bottom: 0;
+    line-height: 1;
+    margin-bottom: 5px;
```

Comment:
The styleguide is supposed to reflect the [UI guidelines](http://prototypes-wtw.net/controls/#forms-fieldsets). Per spec I understand the need for `margin-bottom: 5px`, but I do not understand why we would need the `margin-bottom: 20px` or `line-height: 1`. Can you elaborate on this?